bin/docpub.js

```diff
@@ -10,18 +10,19 @@
 
 const program = require('commander');
 const pkg = require('../package.json');
-const DocpubPipeline = require('../lib/docpub');
+const Docpub = require('../lib/docpub');
 
 program
     .version(pkg.version)
-    .option('-v --verbose', 'verbose output')
-    .option('-p --path <path>', 'path to documents directory (defaults to process directory)');
+    .option('--doc-path <docPath>', 'path to documents directory (defaults to process directory)')
```

Comment:
I really don't have strong opinion on this. My logic was that since we have different paths, then we need to distinguish them, so I added prefixes.

@elliatab @ADHER what do you think?